src/constants/common.ts

```diff
@@ -134,7 +135,7 @@ export const STORAGE_KEYS = {
   transactionsLoaded: 'transactions-loaded',
   transactionsPending: 'transactions-pending',
   transferSendData: 'transfer-send-data',
-  secureLogin: 'secure-login',
+  secureLoginTimeout: 'secure-login-timeout',
```

Comment:
I suggest to leave old version. I see that the new one is better, but having to write migration only because the name is better is rather bad for me.

Reply:
The old value was not used anywhere, it looks like a rename but I just wanted to create a new value for the `secureLoginTimeout` and noticed the unused `secureLogin` value. There's no migration